### ISSUE TITLE
Allow Custom::Secret to accept an input as Value

### ISSUE THREAD

**Issue by ingcsmoreno** [binxio/cfn-secret-provider#8]
I'm working on a cloudformation stack in which I'd very like to be able to store some private keys as SSM SecretString parameters so I can later provision them into EC2 Instances through cfn-init files.

The custom object [cfn_secret_provider.py](https://github.com/binxio/cfn-secret-provider/blob/master/src/cfn_secret_provider.py) seems to be exactly what I need, except for the fact that it does not accept a custom string value as input.

I think adding an optional `content` property into the input json would do the trick. Then on `cfn_secret_provider.py` class if `content` has been given, use that, if not, generate a random string as always. 

The `json` would look something like:

```json
{
  "Type" : "Custom::Secret",
  "Properties" : {
    "Name" : String,
    "Description" : String,
    "Alphabet" : String,
    "Length" : Integer,
    "Content" : String,
    "KeyAlias" : String,
    "ServiceToken" : String,
    "RefreshOnUpdate": Boolean,
    "ReturnSecret": Boolean,
    "Version": String
  }
}
```

This would be a great and useful feature to add, and could keep backward compatibility without major complication.

**Comment by mvanholsteijn** [comment]
The point of the secret providers is that they provide secrets without a need to store them anywhere, except for the SSM Parameter store.

If the implementation supports a clear text  'value', it is no longer a secret.

Why not use the normal [AWS::SSM::Parameter](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ssm-parameter.html)?

**Comment by ingcsmoreno** [comment]
@mvanholsteijn that was my first try, but sadly cloudformation does not support **SecureString** as type for SSM Parameter.

**Comment by mvanholsteijn** [comment]
Thank you for your pull request. I will merge it for you..

**Comment by mvanholsteijn** [comment]
This feature is available in release [v0.10.2](https://github.com/binxio/cfn-secret-provider/commit/202010126fddddc7d5a8d7287fef0bbf922dbd05)
